@xen-orchestra/backups/_cleanVm.js

```diff
@@ -13,6 +13,7 @@ const { limitConcurrency } = require('limit-concurrency-decorator')
 
 const { Task } = require('./Task.js')
 const { Disposable } = require('promise-toolbox')
+const { path: handlerPath } = require('@xen-orchestra/fs')
```

Comment:
Unless I'm mistaken: 

```suggestion
const handlerPath = require('@xen-orchestra/fs/path.js')
```

Reply:
Yes I am.